src/components/Backdrop.tsx

```diff
@@ -0,0 +1,8 @@
+import { Box } from "@mui/material";
```

Comment:
I'm not so sure we need a separate file for just a Box... The most important thing in this file is the styling of the box. But the styling should be managed differently probably.

Reply:
It's a different style of backdrop, from the MUI one. And I am using it in different places, so I decided to create a separate file for it.

Reply:
That's OK. In any cases, we need to centralize/factorize the styles in a common place as noted.